.github/workflows/publish.yml

```
@@ -67,6 +67,7 @@ jobs:
     needs: [publish]
     if: |
       always() &&
+      github.repository == 'dagger/dagger'
```

```suggestion
      github.repository == 'dagger/dagger' &&
```

I think? Unless I'm missing something about the syntax. Same comment for the other lines below that don't have a trailing &&

---

yep, forgot those ones. fixed!